sixMans/sixMans.py

```diff
@@ -437,11 +437,15 @@ async def lobbyInfo(self, ctx: Context):
         game = self._get_game_by_text_channel(ctx.channel)
         player = ctx.message.author
         if game:
-            await game.post_lobby_info()
-            await ctx.message.add_reaction(Strings.WHITE_CHECK_REACT)
+            if game.has_lobby_info():
+                await game.post_lobby_info()
+                await ctx.message.add_reaction(Strings.WHITE_CHECK_REACT)
```

Comment:
I know I added this, but we don't really need to react to the initial message.

Reply:
updated